src/Sortable/SortableListener.php

```diff
@@ -264,6 +264,16 @@ public function postFlush(EventArgs $args)
                                 // Otherwise we fallback to normal object comparison
                                 if ($gr instanceof Comparable) {
                                     $matches = $gr->compareTo($value);
+                                    // @todo: Remove "boolean" check and only support integer as the interface expects.
+                                    if (is_int($matches)) {
+                                        $matches = 0 === $matches;
+                                    } else {
+                                        @trigger_error(sprintf(
+                                            '%1$s::compareTo() method should return an integer. A %2$s was returned instead.',
+                                            Comparable::class,
+                                            gettype($matches)
```

Comment:
```suggestion
                                            'Support for "%s" as return type from "%s::compareTo()" is deprecated since'
                                            .' gedmo/doctrine-extensions 3.x and will be removed in version 4.0. Return "integer" instead.',
                                            gettype($matches),
                                            Comparable::class
```

Reply:
@phansys Done.